sdk/storage/azure-storage-file-datalake/tests/test_file_system.py

```diff
@@ -416,41 +489,46 @@ def test_list_system_filesystems(self, datalake_storage_account_name, datalake_s
         for fs in filesystems:
             if fs.name == "$logs":
                 found = True
-        self.assertEqual(found, True)
+        assert found == True
 
+    @pytest.mark.skip(reason="Feature not yet enabled. Make sure to record this test once enabled.")
     @DataLakePreparer()
-    def test_rename_file_system_with_source_lease(self, datalake_storage_account_name, datalake_storage_account_key):
-        if not self.is_playback():
-            return
+    @recorded_by_proxy
+    def test_rename_file_system_with_source_lease(self, **kwargs):
+        datalake_storage_account_name = kwargs.pop("datalake_storage_account_name")
+        datalake_storage_account_key = kwargs.pop("datalake_storage_account_key")
+
         self._setUp(datalake_storage_account_name, datalake_storage_account_key)
         old_name = self._get_file_system_reference(prefix="old")
         new_name = self._get_file_system_reference(prefix="new")
         filesystem = self.dsc.create_file_system(old_name)
-        filesystem_lease_id = filesystem.acquire_lease()
-        with self.assertRaises(HttpResponseError):
+        filesystem_lease_id = filesystem.acquire_lease(lease_id='00000000-1111-2222-3333-444444444444')
+        with pytest.raises(HttpResponseError):
             self.dsc._rename_file_system(name=old_name, new_name=new_name)
-        with self.assertRaises(HttpResponseError):
+        with pytest.raises(HttpResponseError):
             self.dsc._rename_file_system(name=old_name, new_name=new_name, lease="bad_id")
         new_filesystem = self.dsc._rename_file_system(name=old_name, new_name=new_name, lease=filesystem_lease_id)
-        self.assertEqual(new_name, new_filesystem.get_file_system_properties().name)
+        assert new_name == new_filesystem.get_file_system_properties().name
 
+    @pytest.mark.playback_test_only
```

Comment:
With using the soft delete account from the ARM template, this _should_ be able to run Live now. Same with all soft delete tests.